server/api/eliteApi.js

```diff
@@ -35,7 +39,7 @@ const eliteApiFactory = client => {
   const getBalances = (context, bookingId) => get(context, `api/bookings/${bookingId}/balances`)
   const getCategoryAssessment = (context, bookingId) =>
     get(context, `api/bookings/${bookingId}/assessment/CATEGORY`).catch(map404ToNull)
-  const getCaseLoads = context => get(context, '/api/users/me/caseLoads')
+  const getCaseLoads = context => get(context, 'api/users/me/caseLoads')
```

Comment:
axios didn't care whether we had a double slash in our urls, but superagent does.  I could add the leniency to our client, but not sure I should...

Reply:
might be worth validating/linting config to ensure that the url variables are always in a known format in application code (probably without trailing slashes). I keep getting caught out by some env variables have trailing slashes and others not.